compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -52,6 +52,34 @@ static void createArgs(ArrayRef<OpAsmParser::UnresolvedOperand> operands,
   }
 }
 
+// Verifies that a dispatch |op|'s |workload| matches that of the |exportOp|.
```

Comment:
this is duplicated here and in the stream dialect; today they are the same but as we add support for indirect dispatch they'll differ; the flow level will capture tensors and the stream level will capture resources and we'll want to verify each of those separately.